tests/terraform/runner/resources/example/example.tf

```diff
@@ -467,4 +467,20 @@ resource "aws_s3_bucket" "bridgecrew_cws_bucket" {
   tags = {
     Name = "BridgecrewCWSBucket"
   }
-}
\ No newline at end of file
+}
+
+resource "aws_efs_file_system" "sharedstore" {
+  creation_token                  = "my-product"
+
+  lifecycle_policy {
+    transition_to_ia = "AFTER_30_DAYS"
+  }
+
+    kms_key_id                      = "aws/efs"
+    encrypted                       = True
```

Comment:
@JamesWoolfenden Should be `true`, I believe

Reply:
Yep thanks for finding it. Updated